src/Controls/samples/Controls.Sample.UITests/Issues/Issues15815.xaml.cs

```diff
@@ -0,0 +1,34 @@
+using System;
+using System.Collections.ObjectModel;
+using Microsoft.Maui.Controls;
+using Microsoft.Maui.Controls.Xaml;
+using Microsoft.Maui.Graphics;
+
+namespace Maui.Controls.Sample.Issues
+{
+
+	[XamlCompilation(XamlCompilationOptions.Compile)]
+	[Issue(IssueTracker.Github, 15815, "Horizontal CollectionView does not show the last element under some condition", PlatformAffected.iOS)]
+	public partial class Issues15815 : ContentPage
+	{
+		public Issues15815()
+		{
+			InitializeComponent();
+			col.ItemsSource = new ObservableCollection<ItemViewModel>
```

Comment:
This appears to be a very small list, should we add more items and scroll to ensure that a wide device - like macos does not load them all?

Reply:
baby steps? I tried a test for that but doesn't run reliable with scroll, let's address this issue first.